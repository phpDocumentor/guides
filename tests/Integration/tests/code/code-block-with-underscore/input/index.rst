Underscores in code indented with tabs
======================================

..  code-block:: typoscript

				split {
					token = frontend__
					1.current = 1
					1.wrap = |
				}
