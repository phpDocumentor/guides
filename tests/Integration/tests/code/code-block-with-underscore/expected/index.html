<!-- content start -->
    <div class="section" id="underscores-in-code-indented-with-tabs">
            <h1>Underscores in code indented with tabs</h1>

            <pre><code class="language-typoscript">split {
	token = frontend__
	1.current = 1
	1.wrap = |
}</code></pre>
    </div>

<!-- content end -->
